app/serializers/rest/tag_serializer.rb

```diff
@@ -3,10 +3,12 @@
 class REST::TagSerializer < ActiveModel::Serializer
   include RoutingHelper
 
-  attributes :name, :url, :history
+  attributes :id, :name, :url, :history
 
   attribute :following, if: :current_user?
 
+  delegate :id, to: :object
```

Comment:
I have not written serializers in a while, but I am pretty sure this is superfluous. Adding `:id` to `attributes` should be sufficient.

Reply:
That is accurate. However, I would actually define an `id` method because we most likely want this to be a string.
```suggestion
  def id
    object.id.to_s
  end
```